<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FahrzeugDetail</class>
 <widget class="QWidget" name="FahrzeugDetail">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detailansicht Fahrzeug</string>
  </property>
  <widget class="QLineEdit" name="lineEditKennzeichen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Typ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPreisProTag">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kennzeichen:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Preis pro Tag:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBezeichnung">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTyp">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Bezeichnung:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSchliessen">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Schlie√üen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditBezeichnung</tabstop>
  <tabstop>lineEditKennzeichen</tabstop>
  <tabstop>lineEditPreisProTag</tabstop>
  <tabstop>lineEditTyp</tabstop>
  <tabstop>buttonSchliessen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
